dashboard/app/views/levels/_teacher.html.haml

```diff
@@ -54,3 +57,13 @@
                     %span.puzzle-number
                       = @script_level.position
                 %td.name= link_to student.name, build_script_level_path(@script_level, user_id: student.id, section_id: @section.id)
+  :javascript
+    function resizeScrollable() {
+      var newHeight = $('.teacher-panel').height() -
```

Comment:
ooh, outerHeight() === height() for .teacher-panel ... but innerHeight() is 10px less and I think makes more sense